### PR TITLE
Updated to accommodate UI changes in creating stacks and replicating …

### PR DESCRIPTION
**Pull request by RodneyLester**
*Issue #671:*

*Description of changes:*

I updated the first step, deploy infrastructure, to make it clearer to click "Next" until you are prompted to okay creation of IAM resources.
I updated the second step, configure replication, to have screenshots of the current UI and to use the current text of that UI.

I updated 4 screenshots in the second step.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by RodneyLester** [comment]
Pushing for my mentee to use and closing issues #671 and #615

File: content/Reliability/200_Labs/200_Bidirectional_Replication_for_S3/2_configure_replication.md [hunk 1]
```diff
@@ -19,31 +19,24 @@ Replication is configured via _rules_. There is no rule for bi-directional repli
...
       * **Troubleshooting**: If you get an error saying _The bucket doesn’t have versioning enabled_ then you have chosen the wrong bucket. Double check the bucket name.
-1. Click **Next**
-1. For **IAM Role** select **\<your-naming-prefix\>-S3-Replication-Role-us-east-2** from the search results box
+1. For **IAM Role** leave **Choose from existing IAM roles** selected, and select **\<your-naming-prefix\>-S3-Replication-Role-us-east-2** from the search results box
       * (If you chose a different region as your _east_ region, then look for that region at the end of the IAM role name)
-1. For **Rule name** enter `east to west`
-1. Leave **Status** set to **enabled**
-1. Click **Next**
-1. Review the configuration
+1. For **Encryption**, select **Replicate objects encrypted with AWS KMS** and leave **AWS managed key (aws/s3)** selected.
```
Comment (setheliot) [inline content/Reliability/200_Labs/200_Bidirectional_Replication_for_S3/2_configure_replication.md:34]: Why are you changing this setting?  Used to be leave **Replicate objects encrypted with AWS KMS** _not_ selected -- see old lines 28-32

File: content/Reliability/200_Labs/200_Bidirectional_Replication_for_S3/2_configure_replication.md [hunk 2]
```diff
@@ -86,28 +79,20 @@ After setting up the second rule, you will have completed configuration of bi-di
...
       * **Troubleshooting**: If you get an error saying _The bucket doesn’t have versioning enabled_ then you have chosen the wrong bucket. Double check the bucket name.
-1. Click **Next**
-1. For **IAM Role** select **\<your-naming-prefix\>-S3-Replication-Role-us-west-2** from the search results box
+1. For **IAM Role** leave **Choose from existing IAM roles** selected, and select **\<your-naming-prefix\>-S3-Replication-Role-us-west-2** from the search results box
       * (If you chose a different region as your _west_ region, then look for that region at the end of the IAM role name)
-1. For **Rule name** enter `west to east`
-1. Leave **Status** set to **enabled**
-1. Click **Next**
-1. Review the configuration
+1. For **Encryption**, select **Replicate objects encrypted with AWS KMS** and leave **AWS managed key (aws/s3)** selected.
```
Comment (setheliot) [inline content/Reliability/200_Labs/200_Bidirectional_Replication_for_S3/2_configure_replication.md:90]: Why are you changing this setting? Used to be leave Replicate objects encrypted with AWS KMS not selected -- see old lines 92-96

**Review by setheliot — CHANGES_REQUESTED** [review]
- images should be 1200px wide maximum.. please resize static/Reliability/200_Bidirectional_Replication_for_S3/Images/ReplicatedObject.png
- see other comments about KMS

File: content/Reliability/200_Labs/200_Bidirectional_Replication_for_S3/2_configure_replication.md (hunk 2 above)
Comment (RodneyLester) [reply content/Reliability/200_Labs/200_Bidirectional_Replication_for_S3/2_configure_replication.md:90]: It doesn't work if you do that. Verified that with my mentee. I'll scale the images down.

Comment (RodneyLester) [reply content/Reliability/200_Labs/200_Bidirectional_Replication_for_S3/2_configure_replication.md:90]: Scaled down - take a look

**Review by setheliot — APPROVED** [review]

**Pull request merged**